Private/Test-IsRecentVersion.ps1

```diff
@@ -48,18 +48,18 @@ function Test-IsRecentVersion {
         # Checking the most recent release in GitHub, but we could also use PowerShell Gallery.
         $Uri = "https://api.github.com/repos/trimarcjake/locksmith/releases"
         $Releases = Invoke-RestMethod -Uri $uri -Method Get -DisableKeepAlive -ErrorAction Stop
-        $LatestRelease = $Releases | Sort-Object -Property Published_At -Descending | Select-Object -First 1
```

Comment:
Too much copy/paste. :) You'll need to undo the publish/enable replacements in this function.